css/style.css

```diff
@@ -0,0 +1,84 @@
+.html-bg {
+  background-color: #f4f5f6;
+}
+
+.header-margin {
+  margin-bottom: 10px;
+}
+
+.red-bg {
+  background-color: #f72210;
+}
+
+.red-text {
+  color: #f72210;
+}
+
+#header {
+  height: 100px;
+}
+
+.no-bg {
+  background-color: transparent;
+}
+
+.fs {
+  font-size: 15px;
+}
+
+.container {
+  max-width: 1300px;
+  padding: 0;
+}
+
+.categories {
+  background-color: rgba(225, 225, 255, 0.9);
+  color: #f72210;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.myabsolute {
+  position: absolute;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

Reply:
Hi Bola, am trying to make the position attribute responsive. that is i want to make it